manifest.json

```diff
@@ -35,7 +35,8 @@
     }
   ],
   "background": {
-    "service_worker": "service_worker.js"
+    "service_worker": "service_worker.js",
+    "type": "module"
```

Comment:
The hard coding of the limits bugged me so I changed them to import form web-vitals.js which necessitated this becoming a module.

Reply:
This was completed in #178 instead to make this PR cleaner.